### DISCUSSION TITLE
Lien mort dans les slides html

### DISCUSSION THREAD

**Discussion by Istomine** [web-classroom/web-classroom.github.io#48] [Q&A : Théorie / Cours] [answered]
Bonjour,

Le [lien](https://web-classroom.github.io/slides/2-html-css.html#/3/6) pour la cheat sheet SEO ne marche plus.
Serait-il possible d'avoir une autre référence? 

Merci

**Comment by Nortalle** [comment, marked as answer]
Bonjour !

Merci beaucoup, je pense qu'il n'a même jamais fonctionné...

Je vais le remplacer par https://developers.google.com/search et https://pagespeed.web.dev/

Merci beaucoup

**Reply by Nortalle** [reply]
Je confirme, les nouveaux liens sont en ligne, merci pour votre apport !
